pallets/funding/src/functions.rs

```diff
@@ -134,6 +134,19 @@ impl<T: Config> Pallet<T> {
 		)
 		.map_err(|_| Error::<T>::NotEnoughFundsForEscrowCreation)?;
 
+		// Each project's contribution token requires a deposit on the account that stores the tokens.
+		// We make the issuer pay that deposit for the contribution token treasury account.
+		let contribution_token_treasury_account = <T as Config>::ContributionTreasury::get();
+		let deposit = T::ContributionTokenCurrency::deposit_required(project_id);
+		T::NativeCurrency::transfer(issuer, &contribution_token_treasury_account, deposit, Preservation::Preserve)
+			.map_err(|_| Error::<T>::NotEnoughFundsForCTDeposit)?;
+		T::NativeCurrency::hold(
+			&HoldReason::FutureDeposit(project_id).into(),
+			&contribution_token_treasury_account,
+			deposit,
+		)
```

Comment:
Is this hold actually necessary? These tokens won't go anywhere, so I don't think we need to "hold" them?

Reply:
Just to be super sure that no one is able to spend those PLMC on something else by accident, even If the only way to spend them is with governance

Reply:
Might be a bit overkill. If this "treasury" account is different from the treasury account of the treasury pallet, these tokens would only be able to be spend by governance. But that would be a potentially malicious proposal, so should be blocked by council/TC. If we already talking about malicious proposal, it could probably already include something to remove hold as well. So lets keep it simpel and do not add an (in my eyes) unnecessary hold.